tools/scripts/build_projects.py

```diff
@@ -132,22 +132,6 @@ def to_blue(str):
 
 HW_DIR_NAME = 'hardware'
 NEW_HW_DIR_NAME = 'new_hardware'
-FILE_TO_DOWNLOAD = 'latest.zip'
-
-def download_all_hw(builds_dir):
```

Comment:
we should keep things in modules. Looks like you moved the new download flow inside main function. I'd suggest keeping it in a separate function as it was until now.